app/src/main/res/drawable-xhdpi/ic_error_session_crashed.xml

```diff
@@ -0,0 +1,383 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="300dp"
```

Comment:
300dp x 300dp is getting a bit iffy for a VectorDrawable because of the rasterization performance penalty. We may want to consider PNGs. https://developer.android.com/studio/write/vector-asset-studio.html#when

Reply:
I'm going to resize to 200x200 for now. This is just a placeholder UX anyway, and a slightly smaller image shouldn't be an issue!